### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by xtianae7**
Templates

### CODE REVIEW

**Comment by vicsufer** [comment]
I'm not completely sure but I think issue templates should be under a .github directory, [step 7](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-default-community-health-file#creating-a-repository-for-default-files)

**Comment by xtianae7** [comment]
> @xtianae7 I'm not completely sure but I think issue templates should be under a .github directory, [step 7](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-default-community-health-file#creating-a-repository-for-default-files)

You are completely right, missed that. Thank you!

**Review by vicsufer — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+### Requirements
```
Comment (blalop) [inline PULL_REQUEST_TEMPLATE.md:1]: Not completely sure I'm having a good grasp about this, but should this be in root folder and not under .github?

Comment (vicsufer) [reply PULL_REQUEST_TEMPLATE.md:1]: Github documentation is not completely clear about this, we can try as it is now and if is not working move the PR Template to the root.

**Review by dortegau — APPROVED** [review]

**Review by caldito — APPROVED** [review]

**Pull request merged**